spec/griddler/cloudmailin/adapter_spec.rb

```diff
@@ -1,50 +1,149 @@
-    expect(normalized_params[:attachments]).to eq [upload_1, upload_2]
+  it 'normalizes parameters where the recipient has been CCed' do
+    expect(normalized(cc_params)).to be_normalized_to(
+      cc: ['Some Identifier <some-identifier@example.com>'],
+      to: ['emily@example.com'],
+      from: 'Joe User <joeuser@example.com>',
+      subject: 'Re: [ThisApp] That thing',
+      text: /Dear bob/
+    )
+  end
+
+  it 'changes attachments to an array of files' do
+    params = default_params.merge(attachments: { '0' => upload_1, '1' => upload_2 })
+
+    normalized_params = normalized(params)
+    expect(normalized_params[:attachments]).to include(upload_1, upload_2)
+    expect(normalized_params[:attachments]).not_to include('0')
+    expect(normalized_params[:attachments]).not_to include('1')
   end
 
   it 'has no attachments' do
-    params = default_params
+    expect(normalized[:attachments]).to be_empty
+  end
+
+  it 'wraps to in an array' do
+    expect(normalized[:to]).to eq([default_params[:headers][:To]])
+  end
+
+  it 'wraps cc in an array' do
+    expect(normalized[:cc]).to eq([default_params[:headers][:Cc]])
+  end
+
+  it 'returns the date' do
+    expect(normalized[:date]).to eq(default_params[:headers][:Date].to_datetime)
+  end
 
-    normalized_params = Griddler::Cloudmailin::Adapter.normalize_params(params)
-    expect(normalized_params[:attachments]).to be_empty
+  it 'returns an array even if cc is empty' do
+    expect(normalized(nocc_params)[:cc]).to eq([])
   end
 
+  def normalized(params = default_params)
+    Griddler::Cloudmailin::Adapter.normalize_params(params)
+  end
+
+  # There are some example real-world headers from Cloudmailin
+  # in doc/real_world_headers.rb
   def default_params
     {
-      envelope: { to: 'Some Identifier <some-identifier@example.com>', from: 'Joe User <joeuser@example.com>' },
-      headers: { Subject: 'Re: [ThisApp] That thing', Cc: 'emily@example.com' },
-      plain: <<-EOS.strip_heredoc.strip
-        Dear bob
+      envelope: {
+        to: 'some-identifier@example.com',
+        from: 'joeuser@example.com'
+      },
+      headers: {
+        Subject: 'Re: [ThisApp] That thing',
+        From: 'Joe User <joeuser@example.com>',
+        To: 'Some Identifier <some-identifier@example.com>',
+        Cc: 'emily@example.com',
+        Date: 'Fri, 30 Sep 2016 10:30:15 +0200'
+      },
+      plain: "Dear bob\n\nReply ABOVE THIS LINE\n\nhey sup"
+    }
+  end
 
-        Reply ABOVE THIS LINE
+  def bcc_params
+    default_params.merge(
+      envelope: default_params[:envelope].merge(to: 'sandra@example.com')
+    )
+  end
 
-        hey sup
-      EOS
-    }
+  def nocc_params
+    default_params.merge(
+      headers: default_params[:headers].except(:Cc)
+    )
+  end
+
+  def cc_params
+    default_params.merge(
+      headers: default_params[:headers].merge(
+        Cc: 'Some Identifier <some-identifier@example.com>',
+        To: 'emily@example.com'
```

Comment:
It may be that hound is using master rather than the PR's details. I think I've seen this before. Happy to merge as the only issues are quoted strings and commas.